MCDataProducts/inc/ProcessCode.hh

```diff
@@ -87,6 +87,7 @@ namespace mu2e {
       mu2eExternalRMC,         mu2eFlateMinus,      mu2eFlatePlus, mu2eFlatPhoton, // 175
       mu2eunused1, mu2eunused2, mu2eunused3, mu2eunused4, 
       mu2eunused5, mu2eunused6, mu2eunused7, mu2eunused8, 
+      mu2eCePlusLeadingLog,
```

Comment:
Questions on the addition of the mu2eCePlusLeadingLog process code:

1) the SU2020 sims did generate mu- to e+ at leading log already, what code did they use?  Do we want to change that?

2) Should new process codes replace mu2eunused1 etc, or should they be addded after as is done here?

I do not have a strong opinion on either of these questions, but others might.

Reply:
SU2020 used tuple-based stops, it is not reusable.  That code is in Offline/main anyways, since years.  The point of this PR is to replace it.
I prefer to infill the 'unused' codes before extending the enum.

Reply:
I prefer that we define new enum values rather than re-use retired ones.  Also, I propose the following policy.  If you want to extend an enum on Mu2eII_SM21 you must first extend it on main and then either merge or cherry-pick the change into Mu2eII_SM21.   If not we can get into the position that the same value has different meanings on different branches.   This particular PR is aimed at main, so my comment is not important here.

Reply:
@brownd1978 OK., do I will replace the "unused1" with my new code